social_website/elastic_search.py

```diff
@@ -22,6 +22,18 @@ def get_related_collections(collection):
                                 }
                   }
         }
+    if featured:
+        q ={"query": {
+                        "bool" : {
+                                  "must_not" : {"term" : { "uid" : collection.uid }},
+                            "should" : [
+                                        { "term": { "featured": True }},
+                                        
```

Comment:
Can we include other Should clauses? I assume that Should is distinct from Must.